study-builder/studies/testboston/study-events.conf

```diff
@@ -79,6 +79,22 @@
       "dispatchToHousekeeping": false,
       "order": 3
     },
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": ${id.act.address},
+        "statusType": "COMPLETE"
+      },
+      "action": {
+        "type": "ACTIVITY_INSTANCE_CREATION",
+        "activityCode": ${id.act.symptoms_survey}
+      },
+      "cancelExpr": """user.studies["testboston"].forms["SYMPTOMS_SURVEY"].hasInstance()""",
+      "maxOccurrencesPerUser": 1,
+      "delaySeconds" : 259200 #72hrs
```

Comment:
Expected this to delay activity instance creation by 72 hours, but doesn't seem to work !